### PR TITLE
Add Dock Manager customization topic

### LINKED ISSUE #933: Create WC DockManager customization topic.
It should include information about recent new features/functionality. Include info for the following properties/methods: 

focusPane()
showPaneHeaders
acceptsInnerDock
allowInnerDock
containedInBoundaries

### PR DESCRIPTION
**Pull request by onlyexeption**
Closes #933

### CODE REVIEW

File: doc/en/components/layouts/dock-manager-customization.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+title: {Platform} Dock Manager | Customization | Infragistics
+_description: Use Infragistics' {Platform} dock manager component to manage the layout through panes, with the ability to customize it. Check out {ProductName} dock manager tutorials!
+_keywords: dock manager, layout, customization, {ProductName}
+mentionedTypes: ['DockManager']
```
Comment (mtsvyatkova) [inline doc/en/components/layouts/dock-manager-customization.md:5]: Add 'ContentPane' as mentioned type

File: doc/en/components/layouts/dock-manager-customization.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+---
+
+### Customizing {Platform} Dock Manager
```
Comment (mtsvyatkova) [inline doc/en/components/layouts/dock-manager-customization.md:8]: This section's title needs to be the same size as the others in the topic: '## Customizing {Platform} Dock Manager'

File: doc/en/components/layouts/dock-manager-customization.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+2. **Unpinned Panes**: When the targeted pane is unpinned, `focusPane` will flyout the pane to its open state.
+
+3. **Regular Pinned Panes**: For regular pinned panes, `focusPane` will set the pane as the `activePane`.
+
+
+By utilizing this method, developers can effortlessly control the visibility and positioning of panes based on user interaction or application events.
+
+Try it for yourself in the sample below:
+
+### Demo
```
Comment (mtsvyatkova) [inline doc/en/components/layouts/dock-manager-customization.md:36]: We could skip the 'Demo' subtitles as they are repeating in the left-side navigation menu. In the other dock manager topics we are displaying samples without any subtitle.

File: doc/en/components/layouts/dock-manager-customization.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+By setting the `acceptsInnerDock` property of an `IgcContentPane`, developers can control docking inside specific content panes. When this property is set to false, users will be restricted from performing inner docking in the specified pane.
+
+
+```ts
+{
+    type: IgcDockManagerPaneType.contentPane,
+    header: 'Floating 1',
+    contentId: 'content3'
+    contentId: 'content3',
```
Comment (mtsvyatkova) [inline doc/en/components/layouts/dock-manager-customization.md:60]:
```suggestion
    contentId: 'content3',
```

**Review by mtsvyatkova — CHANGES_REQUESTED** [review]

**Review by ChronosSF — DISMISSED** [review]

**Review by mtsvyatkova — APPROVED** [review]

File: doc/en/components/layouts/dock-manager-customization.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+By utilizing this method, developers can effortlessly control the visibility and positioning of panes based on user interaction or application events.
+
+Try it for yourself in the sample below:
+
+`sample="/layouts/dock-manager/focus-panes", height="600", alt="{Platform} Focus panes programmatically in Dock Manager Example"`
+
+
+## Auto-hide Pane Headers
+
+With the `showPaneHeaders` property of the DockManager, developers now have the flexibility to control the visibility of pane headers within the `layout`. By default, `showPaneHeaders` is set to **true** ensuring that pane headers are always visible. When set to **false**, all pane headers will be hidden until you hover your mouse over the top edge of a content pane. The corresponding pane header will appear and it will smoothly hide once your mouse moves away. Take a look at the example below:
```
Comment (igdmdimitrov) [inline doc/en/components/layouts/dock-manager-customization.md:64]: Update this section - `showPaneHeaders` was later modified to be either `'onHoverOnly'` or `'always'`

**Pull request merged**